### PR TITLE
chore(portal): Show support options on the billing page and remove features table

### PR DESCRIPTION
**Pull request by AndrewDryga**
Closes https://github.com/firezone/gtm/issues/237

Screenshot 2024-04-09 at 12 15 53

### CODE REVIEW

File: elixir/apps/web/lib/web/live/settings/billing.ex [hunk 1]
```diff
@@ -166,32 +178,51 @@ defmodule Web.Settings.Billing do
...
+                )
+              }
+            >
+              an email
+            </.link>
+            and we will get back to you as soon as possible.
+          </span>
+
+          <span :if={@account.metadata.stripe.support_type == "email_and_slack"}>
+            Please send us a message in the shared Slack channel or <.link
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/settings/billing.ex:204]: @jamilbk please lmk if you have better ideas for the content here

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 2]
```diff
@@ -1108,7 +1108,7 @@ defmodule Web.CoreComponents do
 
   def feature_name(%{feature: :flow_activities} = assigns) do
     ~H"""
-    See detailed flow activities <span>(beta)</span>
+    See detailed Resource access logs <span>(beta)</span>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/core_components.ex]:
```suggestion
    See detailed Resource access logs
```

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 3]
```diff
@@ -1120,7 +1120,7 @@ defmodule Web.CoreComponents do
 
   def feature_name(%{feature: :traffic_filters} = assigns) do
     ~H"""
-    Filter traffic using protocol and port rules <span>(beta)</span>
+    Restrict access based on port and protocol rules <span>(beta)</span>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/core_components.ex]:
```suggestion
    Restrict access based on port and protocol rules
```

File: elixir/apps/web/lib/web/components/core_components.ex [hunk 4]
```diff
@@ -1130,6 +1130,12 @@ defmodule Web.CoreComponents do
     """
   end
 
+  def feature_name(%{feature: :rest_api} = assigns) do
+    ~H"""
+    REST API <span>(beta)</span>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/components/core_components.ex]:
```suggestion
    REST API
```

File: elixir/apps/web/lib/web/live/settings/billing.ex (hunk 1 above)
Comment (jamilbk) [reply elixir/apps/web/lib/web/live/settings/billing.ex:204]: I was going to just suggest entering the Slack channel URL directly, then you can link directly to it here.

I think all plans should have email support option, we'll just prioritize using HubSpot.

So you could just have a metadata field `slack_url` or something

**Review by jamilbk — APPROVED** [review]

**Pull request merged**